src/Features/Core/Portable/Diagnostics/HostAnalyzerManager.cs

```diff
@@ -166,9 +166,12 @@ public bool IsAnalyzerSuppressed(DiagnosticAnalyzer analyzer, Project project)
                 return false;
             }
 
+            // don't capture project
+            var projectId = project.Id;
```

Comment:
Can this file change be broken into another PR?  I'm fine with this change, but i think reviewing the other change will take longer.